tests/gis_tests/geoapp/test_serializers.py

```diff
@@ -70,6 +70,14 @@ def test_srid_option(self):
             [int(c) for c in geodata['features'][0]['geometry']['coordinates']],
             [1564802, 5613214])
 
+    @mock.patch('django.contrib.gis.serializers.geojson.HAS_GDAL', False)
+    def test_without_gdal(self):
+        # Without coordinate transformation, the serialization should succeed:
+        serializers.serialize('geojson', City.objects.all())
+        with self.assertRaises(serializers.base.SerializationError):
```

Comment:
like to use `assertRaisesMessage` to ensure this is the SerializationError we expect.